src/main.rs

```diff
@@ -85,7 +85,30 @@ fn run() -> Result<(), MainError> {
         }
         cli::Command::Install(args) => handle_passed_through("install", args),
         cli::Command::Update(args) => handle_passed_through("update", args),
-        cli::Command::Other(args) => pass_to_sub(args.join(" ")),
+        cli::Command::Other(args) => {
+            let mut cwd = env::current_dir().expect("Couldn't find the current working directory");
+            if cli.workspace_root {
+                cwd = workspace::find_workspace_root(&cwd)?;
+            }
```

Comment:
Reduce unnecessary mutation.

```suggestion
            let cwd = env::current_dir().expect("Couldn't find the current working directory");
            let cwd = if cli.workspace_root {
                workspace::find_workspace_root(&cwd)?
            } else {
                cwd
            };
```

Reply:
Nevermind, it appears @zkochan prefers the mutation style as shown in the above code block.